NeoML/src/common.h

```diff
@@ -18,5 +18,8 @@ limitations under the License.
 #include <NeoMathEngine/Platforms.h>
 
 #ifdef NEOML_USE_FINEOBJ
+#ifndef NOMINMAX
```

Comment:
Why this is inside `NEOML_USE_FINEOBJ`?
Isn't it better to wrap it into
`#if FINE_PLATFORM(FINE_WINDOWS) && !defined(NOMINMAX)` outside of `#ifdef NEOML_USE_FINEOBJ`?

Reply:
Fixed.
Put the NOMINMAX outside any other defines